webapp/certified/views.py

```diff
@@ -264,10 +265,21 @@ def certified_home():
 
     if request.args:
         query = request.args.get("q", default=None, type=str)
+
+        # Old site replacements
+        if set(request.args) & set(["query", "vendors"]):
+            # Convert ImmutableMultiDict into normal dict
+            old_params = request.args.to_dict()
```

Comment:
If you're gonna manipulate this list in-place to replace old params with new ones, then `old_params` isn't a terribly accurate name. `params` would be better. Or even `parameters`.